src/components/checkbox/checkbox.spec.js

```diff
@@ -298,14 +298,14 @@ describe('mdCheckbox', function() {
       expect(isChecked(checkbox)).toBe(false);
     });
 
-    it('should submit a parent form when ENTER is pressed', function () {
+    it('should not submit a parent form when ENTER is pressed', function () {
       var form = compileAndLink('<form><md-checkbox ng-checked="value"></md-checkbox></form>');
       angular.element(form[0].getElementsByTagName('md-checkbox')[0]).triggerHandler({
         type: 'keypress',
         keyCode: $mdConstant.KEY_CODE.ENTER
       });
       pageScope.$apply();
-      expect(form[0].classList.contains('ng-submitted')).toBe(true);
+      expect(form[0].classList.contains('ng-submitted')).toBe(false);
     });
 
```

Comment:
You might want to add a test to prove that a checkbox is submitted when expected as well.